StreamCat.py

```diff
@@ -102,30 +102,30 @@
     Connector = f"{OUT_DIR}/{row.FullTableName}_connectors.csv"
     catTime = dt.now()
     for zone, hydroregion in INPUTS.items():
-        if not os.path.exists(f"{OUT_DIR}/{row.FullTableName}_{zone}.csv"):
-            pre = f"{NHD_DIR}/NHDPlus{hydroregion}/NHDPlus{zone}"
-            if not row.accum_type == "Point":
-                izd = f"{mask_dir}/{zone}.tif" if mask_dir else f"{pre}/NHDPlusCatchment/cat"
-                cat = createCatStats(
-                    row.accum_type,
-                    LL,
-                    izd,
-                    OUT_DIR,
-                    zone,
-                    row.by_RPU,
-                    mask_dir,
-                    NHD_DIR,
-                    hydroregion,
-                    apm,
-                )
-            if row.accum_type == "Point":
-                izd = f"{pre}/NHDPlusCatchment/Catchment.shp"
-                cat = PointInPoly(
-                    points, zone, izd, pct_full, mask_dir, apm, summaryfield
-                )
-            cat.to_csv(f"{OUT_DIR}/{row.FullTableName}_{zone}.csv", index=False)
-            in2accum = len(cat.columns)
-    print("Cat Results Complete in : " + str(dt.now() - catTime))
+    #     if not os.path.exists(f"{OUT_DIR}/{row.FullTableName}_{zone}.csv"):
```

Comment:
this isn't going to work. Why don't we want to do cat stats here? Did you spin up the cat stat files yourself for slope? as long as they are at eh path above this section would get skipped anyway

Reply:
This was just me forgetting to uncomment section of code when I was doing a custom run - pushed up commit uncommenting that section.